### ISSUE TITLE
Missing Icon For Radiance

### ISSUE THREAD

**Issue by bendangelo** [nvbn/everpad#89]
I use the radiance theme on Ubuntu. Currently the everpad icon is still white like its on the ambiance theme. Would be nice if there is a dark icon too for radiance.

**Comment by acutbal** [comment]
Hello!!

I used also a similar clear theme as you, and also, had the same problem as you. 

The solution is very easy... 

http://ubuntuone.com/51FREgzIzUps7NDQK1iZn2

Do a back up of original icons placed in:

1.- /usr/share/icons/hicolor/64x64/apps/everpad-mono.png
2.- /usr/share/pixmaps/everpad-mono.png

An then, change them by the one linked before -->  http://ubuntuone.com/51FREgzIzUps7NDQK1iZn2

Finally, logout o restart, do what you prefer.

I inverted the color of the original icon with Gimp.

Hope help, best regards!!
